mono/metadata/sysmath.c

```diff
@@ -222,7 +222,12 @@ ves_icall_System_Math_ScaleB (gdouble x, gint32 n)
 gdouble
 ves_icall_System_Math_FusedMultiplyAdd (gdouble x, gdouble y, gdouble z)
 {
+#ifdef TARGET_WASM
+	/* Enabling these leads to error: undefined symbol: llvm_fma_f32 from esmcripten */
+	g_assert_not_reached ();
```

Comment:
`return x * y + z`?

Reply:
I thought it might be close enough. Agreed.

Reply:
The float version of `FusedMultiplyAdd` can be emulated with doubles though:
`return ((float)((double)x) * ((double)y) + ((double)z));`